steps/zulip-notifications/0.1.1/step.yml

```diff
@@ -0,0 +1,289 @@
+    summary: The email address for your bot
+    title: Zulip Bot Email Address
+  zulip_step_bot_email: null
+- opts:
+    description: |
+      This variable holds the value of the bot's api key that you want the
+      messages to originate from. While you could technically enter your own password
+      in this field, __this is **_STRONGLY_** discoraged__.
+
+      To create a new bot:
+        1. From the desktop (main page), click on the `gear` icon in the upper
+          right corner.
+        2. Select `Settings`.
+        3. On the left, click `Your bots`.
+        4. Select `Add a new bot`.
+        5. For the `Bot type`, you want `Generic bot`
+        6. Enter whatever values you would like for the `Name`, `Username`, and
+          `Profile picture` fields.
+        7. Select `Create bot`, this will take you back to the `Your bots` page.
+        8. Under the bot you just created, the email address you need is the `API KEY`.
+
+      To find your bot's API Key for an existing bot:
+        1. From the desktop (main page), click on the `gear` icon in the upper
+          right corner.
+        2. Select `Settings`.
+        3. On the left, click `Your bots`.
+        4. Under the bot you want to use, the email address you need is the `API KEY`.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: The API key for your bot
+    title: Zulip Bot API Key
+  zulip_step_bot_key: null
+- opts:
+    description: "This variable holds the list of reciepients that you want to send
+      build\nnotifications to for non pull request builds. The recipients can be either\nstreams
+      (i.e. `Builds`) or private messages (i.e. \n`some-person@diolapp.zulipchat.org`).\n\n**IMPORTANT
+      INFORMATION**\nYou can **_only_** send messages to streams that are a part of
+      **_your_**\ndomain. This is not a limitation imposed by this step, it is a limitation\nimposed
+      by Zulip. As far as I know, as of April 2, 2019, you can send a\nprivate message
+      to any user, so long as you know their email address.\n\nTo figure out your
+      domain, simply look at the URL you visit. If the URL is\n`https://mySuperCoolOrganization.zulipchat.org`,
+      then your domain is\n`mySuperCoolOrganization`.\n\nTo have multiple reciepients,
+      all you need to do is separate them with a comma\n`,`, space around the comma
+      is entirely optional. You can also mix and match streams\nand private messages.\n\n```\nExamples:\n\n
+      \ Valid:  \"Stream1, some-user@someOrg.zulipchat.com, Stream2\"\n  Valid:  \"Stream1,some-user@someOrg.zulipchat.com,Stream2\"\n
+      \ Valid:  \"Stream1 , some-suer@someOrg.zulipchat.com , Stream2\"\n```\n"
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: List of recipients to send build notifications to
+    title: Zulip Recipeients
+  zulip_step_recipients: null
+- opts:
+    content: '{buildEmoji} Build {buildNumber} has {buildResult}!'
```

Comment:
The build failed when using previous cli version, due to unknown key: `error: not supported key found in options: content`
(https://github.com/bitrise-io/bitrise/releases/download/1.24.0/bitrise-Darwin-x86_64)

Reply:
So the content tag is not supported on older versions of Bitrise? How would I enable a multiline input with preserved newlines then, like the script step?

Reply:
You can do it just like the script step: https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/script/1.1.5/step.yml#L42-L65 :)